JpaLayer/src/main/java/com/axinalis/consumer/TransactionsConsumer.java

```diff
@@ -0,0 +1,38 @@
+package com.axinalis.consumer;
+
+import com.axinalis.entity.TransactionEntity;
+import com.axinalis.model.Transaction;
+import com.axinalis.repository.TransactionRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransactionsConsumer {
+
+    private TransactionRepository repository;
+    private ObjectMapper mapper;
+
+    public TransactionsConsumer(@Autowired TransactionRepository repository,
+                                @Autowired ObjectMapper mapper) {
+        this.repository = repository;
+        this.mapper = mapper;
+    }
+
+    @KafkaListener(topics = "transactions")
+    public void listen(Transaction message) {
+        TransactionEntity transactionToDb = new TransactionEntity();
+
+        transactionToDb.setBank(message.getBank());
```

Comment:
Also could be off-top related to Kafka course: but I would recommend you to read about Layered Architecture and DAO layer (probably best option to move transaction persistence into DAO):
https://www.javaguides.net/2020/06/best-practice-to-develop-persistence-or-dao-layer.html
https://stackoverflow.com/questions/13785634/responsibilities-and-use-of-service-and-dao-layers

(for this particular course it's not so critical, but still recommend to check).